lib/prototype-admin/reset.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'layout.html' %}
+
+{% block pageTitle %}
+  Reset data - NHS prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {{ backLink({
+      "text": "Back",
+      "href": backHref
+    })
+  }}
```

Comment:
Minor indenting tweak here

```suggestion
  {{ backLink({
    "text": "Back",
    "href": backHref
  }) }}
```